cluster/saltbase/salt/kube-proxy/kube-proxy.manifest

```diff
@@ -38,6 +38,14 @@ kind: Pod
 metadata:
   name: kube-proxy
   namespace: kube-system
+  # This annotation lowers the possibility that kube-proxy gets evicted when the
+  # node is under memory pressure, and prioritizes it for admission, even if
```

Comment:
Hmm, I don't think anything in the kubelet reads the disk pressure node condition, does it? 

if we're out of disk we will still not admit it (https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet.go#L1677) and if we admit it in that case it'll probably make the disk situation worse? 

The current fix is only around the admit method of the eviction manager, and only based on the NodeMemoryPressure condition.